src/client.rs

```diff
@@ -10,7 +10,9 @@ use serde_json::Value;
 use tokio::prelude::*;
 
 use crate::protocol;
-use crate::protocol::{FastMessage, FastMessageData, FastMessageStatus, FastParseError};
+use crate::protocol::{
+    FastMessage, FastMessageData, FastMessageServerError, FastMessageStatus, FastParseError,
```

Comment:
I despise these trailing commas that `rustfmt` adds. Fortunately this will be [configurable](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#trailing_comma) in a future version of `rustfmt`. I'm curious on your thoughts on them. I've been manually removing them, but I don't want to constantly be fighting against `rustfmt` so maybe I'll just wait until the configuration option is available and remove them all then.

Reply:
I tend to agree, but I don't feel that strongly about it.  I'm glad to see `trailing_comma` coming to `rustfmt`.